Project.toml

```diff
@@ -18,7 +18,7 @@ Roots = "f2b01f46-fcfa-551c-844a-d8ac1e96c665"
 [compat]
 DataStructures = "0.17"
 DiffEqBase = "6"
-DiffEqProblemLibrary = "4.3"
+DiffEqProblemLibrary = "4"
```

Comment:
That's not useful, the tests will certainly error for any version prior to 4.3. IMO the main issue is that we test on Julia 1.0 on Travis, but all newer versions of DiffEqBiological require Julia 1.1: https://github.com/JuliaDiffEq/DiffEqBiological.jl/blob/master/Project.toml#L22

Reply:
IMO we should just run the CI tests on Julia >= 1.1. Or split the DiffEqProblemLibrary into different packages for DDEs, SDEs, and so on, then we do not depend on DiffEqBiological anymore (I guess)...

Reply:
Oh, I didn't realize Travis was running 1.0...

Reply:
```suggestion
DiffEqProblemLibrary = "4.3"
```